server/src/main/java/org/eclipse/lsp/cobol/service/CobolTextDocumentService.java

```diff
@@ -431,6 +434,27 @@ private void notifyAnalysisFinished(
             .build());
   }
 
+  private Map<String, List<Location>> extractCopybookUsages(AnalysisResult result) {
```

Comment:
The `notifyAnalysisFinished` method uses only a list of copybook URIs, so this logic can be simplified.